tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -283,9 +300,11 @@ struct computeExpected<InType, OutType, ShaderOpKind::WaveActiveMax> {
 
 template <typename InType, typename OutType>
 struct computeExpected<InType, OutType, ShaderOpKind::WaveActiveMin> {
-  OutType operator()(const std::vector<InType> &inputs, const std::vector<int> &masks, int maskValue) {
+  OutType operator()(const std::vector<InType> &inputs,
+                     const std::vector<int> &masks, int maskValue,
+                     unsigned int index) {
     OutType minimum = std::numeric_limits<OutType>::max();
-    for (size_t i = 0; i < inputs.size(); ++i) {
+    for (size_t i = 0; i < index; ++i) {
       if (masks.at(i) == maskValue && inputs.at(i) < minimum)

```

Comment:
If you are asking why 'at' instead of '[]', it was to double check if the given index is within the size of the input lists.